modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 //* architecture/installation-.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
 
 [id="registry-configuring-storage-vsphere_{context}"]
 = Configuring registry storage for vSphere
```

Comment:
The same issue like baremetal image registry configuration.
Obviously the above command is using wrong syntax. 
`
$ oc get pod -n configs.imageregistry.operator.openshift.io/cluster
invalid namespace "configs.imageregistry.operator.openshift.io/cluster": [may not contain '/']
`
Here you want to verify if a image registry config - 'cluster' is existing for the following modification, but not pod. am I right?
If you want to verify registry existing or not, should be:
`$ oc get po -n openshift-image-registry`